scripts/cluster

```diff
@@ -15,11 +15,36 @@ if [[ "$1" == "polybft" ]] && ! command -v nc >/dev/null 2>&1; then
   dp_error_flag=1
 fi
 
+# Check if docker-compose is installed
+if [[ "$2" == "--docker" ]] && ! command -v docker-compose >/dev/null 2>&1; then
+  echo "docker-compose is not installed."
+  echo "Manual installation instructions: Visit https://docs.docker.com/compose/install/ for more information."
+  dp_error_flag=1
+fi
+
 # Stop script if any of the dependencies have failed
-if [[ "$dp_error_flag" -eq 1]]; then
+if [[ "$dp_error_flag" -eq 1 ]]; then
   echo "Missing dependencies. Please install them and run the script again."
   exit 1
-fi	
+fi
+
+function showhelp(){
+  echo "Usage: cluster {consensus} [{command}] [{flags}]"
+  echo "Consensus:"
+  echo "  ibft            Start Supernets test environment locally with ibft consensus"
+  echo "  polybft         Start Supernets test environment locally with polybft consensus"
+  echo "Commands:"
+  echo "  stop            Stop the running environment"
+  echo "  destroy         Destroy the running environment"
```

Comment:
Good point, thanks. I added here the `write-logs` command.
I agree it would be better to convert it to a flag in the future, but until that's done here we can state how it currently works today.